pyproject.toml

```diff
@@ -72,8 +72,8 @@ semver = "2.8.1"
 sentry-sdk = "1.28.1"
 simplejson = "3.16.0"
 sqlalchemy = "1.3.24"
-sqlalchemy-searchable = "1.2.0"
-sqlalchemy-utils = "0.34.2"
+sqlalchemy-searchable = "1.4.1"
+sqlalchemy-utils = "0.37.9"
```

Comment:
Is it nessesary to bump these versions at the same time? If possible I would prefer that updates dependencies are in a separate change to make troubleshooting easier.

Reply:
@eradman removed these changes